core-libs/setup/ssr/error-handling/multi-error-handlers/propagating-to-server-error-handler.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable, inject } from '@angular/core';
+import { MultiErrorHandler } from '@spartacus/core';
+import { PROPAGATE_ERROR_TO_SERVER } from '../error-response/propagate-error-response';
+
+/**
+ * Error handler responsible for sending an error to the incoming request in the server.
```

Comment:
* Propagates the given error object to the higher layer in the server.
 *
 * It's meant to propagate errors for example to ExpressJS layer when using SSR
 * or to a Prerendering Worker when using Server Prerendering.
 * Currently, it's provided OOTB only in SSR (not prerendering), in the `CxCommonEngine` class.
 *
 * Note: We need it until Angular implements a proper propagation of async errors 
 * from an app to the the higher layer in the server.
 * For more, see the Angular issue https://github.com/angular/angular/issues/33642
 *